ivy/functional/frontends/torch/non_linear_activation_functions.py

```diff
@@ -16,3 +16,10 @@ def softmax(input, dim=None, dtype=None):
 
 
 softmax.unsupported_dtypes = ("float16",)
+
+
+def gelu(input):
```

Comment:
the actural arguments of torch.nn.functional.gelu are "input" and "approximate='none'", according to (https://pytorch.org/docs/stable/generated/torch.nn.functional.gelu.html). You should include both of these in the code and the test. Thank you.